### PR TITLE
PHP 8.2 Support 🚀

### PR DESCRIPTION
**Pull request by Muffinman**
It would be great to have PHP 8.2 support now this has been officially released.

I have made the assumption to bump your `latest` and `stable` tags to be in sync with the PHP releases.

### CODE REVIEW

**Comment by Muffinman** [comment]
Done :)

File: .github/workflows/packages.yml [hunk 1]
```diff
@@ -16,15 +16,13 @@ jobs:
       matrix:
         include:
           - tag: 'latest'
-            php: '8.1'
+            php: '8.2'
           - tag: 'stable'
-            php: '8.1'
+            php: '8.2'
           - tag: '8.1'
```
Comment (lorisleiva) [inline .github/workflows/packages.yml:22]:
```suggestion
          - tag: '8.2'
            php: '8.2'
          - tag: '8.1'
```

**Review by lorisleiva — APPROVED** [review]
Thanks! 🍺

**Comment by lorisleiva** [comment]
Published on GitHub packages and Docker Hub. 🚀

**Pull request merged**